buildutils/src/dependency-graph.ts

```diff
@@ -277,7 +277,7 @@ commander
     '--no-jupyterlab',
     'Do not include dependency connections TO @jupyterlab org packages nor isolated @jupyterlab org packages'
   )
-  .option('--no-phosphor', 'Do not include @phosphor org packages')
+  .option('--no-phosphor', 'Do not include @lumino org packages')
```

Comment:
Should we have two options here? `--no-phosphor` as a legacy and `--no-lumino` as the new option?